primer/src/Primer/UUIDv4.hs

```diff
@@ -0,0 +1,18 @@
+module Primer.UUIDv4 (nextRandom) where
+
+import Foreword
+
+import Data.UUID.Types (UUID)
+import Data.UUID.Types.Internal (buildFromBytes)
+import System.Random (randomIO)
+
+-- | Generate a pseudo-random 'UUID'.
+--
+-- Note: this effect does __not__ generate cryptographically secure
+-- UUIDs. If you need cryptographically secure 'UUID's, use
+-- the effect of the same name from the @uuid@ package.
+nextRandom :: IO UUID
+nextRandom = do
+  [b0, b1, b2, b3, b4, b5, b6, b7, b8, b9, ba, bb, bc, bd, be, bf] <-
+    replicateM 16 randomIO
+  pure $ buildFromBytes 4 b0 b1 b2 b3 b4 b5 b6 b7 b8 b9 ba bb bc bd be bf
```

Comment:
Another option would be something like `nextRandom = randomIO`, since `UUID` has a `Random` instance' which is essentially the same implementation, I think.

Reply:
yeah, I think I'll switch to `randomIO`

Reply:
Done.